rosdep/base.yaml

```diff
@@ -1028,6 +1028,12 @@ flex:
   opensuse: [flex]
   rhel: [flex]
   ubuntu: [flex]
+flite:
```

Comment:
could you check the suggestions made by github actions? (see in "Files changed")

Reply:
I add `alpine` and `rhel` entry.
I have one question. In github PR template, there is no information about `rhel`.
`rhel` is also a `IF AVAILABLE` entry?
https://github.com/ros/rosdistro/blob/master/.github/pull_request_template.md#links-to-distribution-packages

Reply:
The addition of RHEL instructions is tracked by #32388.
The absence of an official package dashboard for RHEL makes this trickier than other distros.

Reply:
i didnt know the issue #32388.
Thank you.